x-pack/plugin/rollup/src/test/java/org/elasticsearch/xpack/rollup/job/RollupIndexerStateTests.java

```diff
@@ -946,10 +956,11 @@ protected void doNextBulk(BulkRequest request, ActionListener<BulkResponse> next
             assertTrue(indexer.maybeTriggerAsyncJob(System.currentTimeMillis()));
             assertThat(indexer.getState(), equalTo(IndexerState.INDEXING));
             latch.countDown();
+            assertBusy(() -> assertTrue(isFinished.get()));
             ESTestCase.awaitBusy(() -> isFinished.get());
```

Comment:
nitpick: I think this awaitBusy can be removed. If the assertBusy above passes, this will always pass (unless there is some async condition that can change back to false). If the assertBusy fails if will fail the test and this line wont execute.

Reply:
You are correct! Let me fix it up really quick.